hi-ml/src/health_ml/utils/data_augmentations.py

```diff
@@ -0,0 +1,81 @@
+from skimage import color
+import cv2
+import numpy as np
+import torch
+
+
+class HEDJitter(object):
+    """Randomly perturbe the HED color space value an RGB image.
+    First, it disentangled the hematoxylin and eosin color channels by color deconvolution method using a fixed matrix.
+    Second, it perturbed the hematoxylin, eosin and DAB stains independently.
+    Third, it transformed the resulting stains into regular RGB color space.
+    Args:
+        theta (float): How much to jitter HED color space,
+         alpha is chosen from a uniform distribution [1-theta, 1+theta]
+         beta is chosen from a uniform distribution [-theta, theta]
+         the jitter formula is **s' = \alpha * s + \beta**
+    """
+    def __init__(self, theta: float = 0.) -> None:   # HED_light: theta=0.05; HED_strong: theta=0.2
```

Comment:
This comment should be in docu, so that people get a sense of what is light/strong

Reply:
Good point - do we need to add to this PR some docs? @maxilse would be the best person to contribute to some docs on this class

Reply:
Have moved this and tried to clarify as much as possible - is it clearer now?